cohort-parent/pom.xml

```diff
@@ -55,6 +58,40 @@
 		<sonar.moduleKey>${project.groupId}:${project.artifactId}</sonar.moduleKey>
 		<sonar.organization>alvearie</sonar.organization>
 		<sonar.host.url>https://sonarcloud.io</sonar.host.url>
+		
+		<!-- Properties added for REST projects-->
+		<whCompanyName>IBM Corp</whCompanyName>
+
+		<whBldVer>${project.version}</whBldVer>
+		<watson.buildtools.version>1.27</watson.buildtools.version>
+
+		<srcjavaDir>src/main/java</srcjavaDir>
+		<srcjavaTestDir>src/test/java</srcjavaTestDir>
+		<srcjavadocDir>${srcjavaDir}</srcjavadocDir>
+		<srcjavadocTestDir>${srcjavaTestDir}</srcjavadocTestDir>
+		<skipTests>true</skipTests>
```

Comment:
Curious what we're doing with setting skipTests to true?

Reply:
This was pulled over from the example project definition and I think it was an attempt to allow skipping tests on a per-project basis (ala https://maven.apache.org/surefire/maven-surefire-plugin/examples/skipping-tests.html). I don't think this is really needed for our projects at this time (we can still use the maven.test.skip property if we want in our dev envs.). I am going to remove this property and it's references until such time as we determine that it is actually needed.